src/main/java/club/sk1er/patcher/mixins/bugfixes/optifine/RenderGlobalMixin_FixSkyVBOs.java

```diff
@@ -0,0 +1,24 @@
+package club.sk1er.patcher.mixins.bugfixes.optifine;
+
+import net.minecraft.client.renderer.RenderGlobal;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(RenderGlobal.class)
+public class RenderGlobalMixin_FixSkyVBOs {
+    @Dynamic("OptiFine")
+    @ModifyVariable(method = "renderSky(Lnet/minecraft/client/renderer/WorldRenderer;FZ)V", at = @At("STORE"), ordinal = 2)
```

Comment:
Seems this could be replaced by `@Redirect`ing the access to `this.renderDistance` to return `256`. I'd prefer that to a reliance on locals.

Reply:
Sounds good, I've just pushed a commit for that which does this instead. Seems to apply with no issue.